package.json

```diff
@@ -81,6 +81,7 @@
     "prepare": "husky && npm run build",
     "preversion": "npm test",
     "version": "cross-env-shell \"npm version $npm_package_version --no-git-tag-version --ws\" && git add **/package*.json",
+    "postinstall": "node build/eslint-esl.version.js",
```

Comment:
The idea is good. The primary case indeed starts from the ESL update – that is absolutely true. However, it doesn't seem 100% correct or optimal to make the ESL package worry about a separate optional package. It doesn't seem like it should be its job.

There are two options I can suggest to resolve the problem:

1. Use some completely indirect way to run a script located and operating within the `esl-eslint-plugin` from the `post-install` script of ESL. This way, we avoid explicit dependencies in ESL itself.

2. (This option sounds more suitable to me) Notify the user during the lint rule run itself. The message will not be missed in the upgrade log, as it will be posted each time ESLint runs with the ESL plugin.